src/v/config/configuration.cc

```diff
@@ -1142,6 +1142,20 @@ configuration::configuration()
       "maximum number of IO and CPU shares that archival upload can use",
       {.visibility = visibility::tunable},
       1000)
+  , retention_local_target_bytes_default(
+      *this,
+      "retention_local_target_bytes_default",
+      "Local retention size target for partitions of topics with cloud storage "
+      "write enabled",
+      {.visibility = visibility::tunable},
+      std::nullopt)
+  , retention_local_target_ms_default(
+      *this,
+      "retention_local_target_ms_default",
+      "Local retention time target for partitions of topics with cloud storage "
+      "write enabled",
+      {.visibility = visibility::tunable},
```

Comment:
can those properties be runtime settable ?

Reply:
Good think you asked. While looking into it I noticed that these defaults are not hooked up properly. I'll fix this and add some more tests.

Reply:
Fixed it and made the properties runtime settable.